feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/README.md

```diff
@@ -0,0 +1,53 @@
+# gNMI-1.3: Benchmarking: Drained Configuration Convergence Time
+
+## Summary
+
+Measure performance of drained configuration being applied.
+
+## Procedure
+
+Configure DUT with maximum number of IS-IS adjacencies, and BGP
```

Comment:
Should we clarify the max number across vendors here? Also, it is unclear about the test failure criteria. Should we also define a performance expectation so that we can better use the test result?

@robshakir @sachendras @xw-g @vosipchu thoughts?

Reply:
@vcheny , as per testcase description in WBB testplan, we need to establish adjacency on physical interfaces.
Number of sessions depends on max number of physical interfaces provided in binding file.

Reply:
Also route count is made as variable.

Reply:
For the pass/fail part:
@vcheny, gNMI-1.3 is just a benchmarking test to collect time taken for convergence in different test scenarios for  BGP and IS-IS. The pass/fail criteria includes
1. Are all the BGP peering and IS-IS adjacencies established?
2. When overload bit is set and metric change set, are all the neighbors on ATE receiving the  change
3. Same for BGP, when the AS_path prepending and MED changed, are all the peers on the ATE receiving them

I see that the test is missing "3" and Overload-bit in "2". Not sure why. But otherwise, the test seems to be doing the right things from the verification standpoint and collecting the time delta for benchmark.

Reply:
@sachendras , I have added BGP benchmarking and ISIS Overload bit testcode back. Please check.